cmd/dao_tests/pubkey_test.go

```diff
@@ -29,8 +31,9 @@ func createPk() *models.Pubkey {
 
 func InitTestEnvironment() error {
 	config.Initialize()
+	logging.InitializeStdout()
```

Comment:
This line actually silences the logging depending on the setting in `local.yaml`. You can enable or disable it, by default it is disabled tho which is much cleaner on CI.

In the other PR I want to take a look on saving all logs into buffer and showing it only on failures.